### DISCUSSION TITLE
Cannot upload bash scripts

### DISCUSSION THREAD

**Discussion by kalebers** [community/community#88550] [Repositories] [answered]
Hello everyone,

I'm facing a little issue while uploading bash scripts. The error message appears when dropping the file 
`Something went really wrong, and we can’t process that file. `

Using GitBash the file doesn't upload to my repository.

I tried to use different browsers and computers, but the same issue appears. 

How can I upload this file?

**Comment by cham1ndux** [comment, marked as answer]
Several factors may contribute to that error. It could be due to a problem with your network connection or an effect on your AV. You can try using [hub](https://github.com/mislav/hub) to avoid it. Or to upload using a VPN or disable your firewall and try uploading.
